components/description-list/README.md

```diff
@@ -0,0 +1 @@
+# Description List
```

Comment:
I just verified locally, it looks like the grid approach only supports 1-to-1 `dt`/`dd` pairing, right? Will be worth calling that out in the readme when you've had time to write one up.